src/mdbExtensionController.ts

```diff
@@ -71,37 +77,50 @@ export default class MDBExtensionController implements vscode.Disposable {
     );
     this._helpExplorer = new HelpExplorer();
     this._playgroundsExplorer = new PlaygroundsExplorer();
+    this._playgroundResultViewProvider = new PlaygroundResultProvider(
+      context,
+      this._connectionController
+    );
+    this._activeConnectionCodeLensProvider = new ActiveConnectionCodeLensProvider(
+      this._connectionController
+    );
+    this._partialExecutionCodeLensProvider = new PartialExecutionCodeLensProvider();
     this._playgroundController = new PlaygroundController(
       context,
       this._connectionController,
       this._languageServerController,
       this._telemetryController,
-      this._statusView
+      this._statusView,
+      this._playgroundResultViewProvider,
+      this._activeConnectionCodeLensProvider,
+      this._partialExecutionCodeLensProvider
     );
     this._editorsController = new EditorsController(
       context,
       this._connectionController,
       this._playgroundController,
       this._statusView,
-      this._telemetryController
+      this._telemetryController,
+      this._playgroundResultViewProvider,
+      this._activeConnectionCodeLensProvider,
+      this._partialExecutionCodeLensProvider
     );
     this._webviewController = new WebviewController(
       this._connectionController,
       this._telemetryController
     );
+    this._editorsController.registerProviders();
```

Comment:
Not related to this ticket, I wonder if we should move calling the `activate` method into the constructor, so that the `extension.js` doesn't create the `mdbExtensionController` and then immediately activate it. Maybe we're doing extra steps there.

Reply:
The `registerProviders` function just wraps the registration of all providers from the constructor, where they were placed before. Moving `registerProviders to the `activate` function causes an error about trying to register an already registered scheme in tests because in tests we do this:

https://github.com/mongodb-js/vscode/blob/master/src/test/suite/index.ts#L45

in addition to this (that is also being called):

https://github.com/mongodb-js/vscode/blob/master/src/extension.ts#L41